lib/ReportHistoryStore.jsx

```diff
@@ -260,7 +289,7 @@ export default class ReportHistoryStore {
         })
             .done((recentHistory) => {
                 // Update history with new items fetched
-                this.mergeHistoryByTimestamp(reportID, recentHistory);
```

Comment:
Is this method now unused?

Reply:
No we're keeping it for this https://github.com/Expensify/expensify-common/blob/dfdfd3082130797420ff7e416d21227b1b5a4172/lib/ReportHistoryStore.jsx#L130